src/Annotations/MuatationNamespace.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TheCodingMachine\GraphQLite\Annotations;
+
+use Attribute;
+
+/**
+ * @Annotation
+ * @Target({"CLASS"})
+ * @Attributes({
+ *   @Attribute("name", type = "string"),
+ * })
+ */
+#[Attribute(Attribute::TARGET_CLASS)]
+class MuatationNamespace
```

Comment:
```suggestion
class MutationNamespace
```

Reply:
Thanks, for some reason i keep writing `Muatation` :smile: